edi_sale_ubl_oca/tests/test_order_in_full_flow.py

```diff
@@ -48,6 +56,7 @@ def test_new_order(self):
         ack_exc_record = order.exchange_record_ids.filtered(
             lambda x: x.type_id == self.exc_type_out
         )
+        ack_exc_record.action_exchange_generate()
```

Comment:
Could you explain a little bit, why we have set `exchange_file_auto_generate` already, but still need to call `action_exchange_generate` here (just asking)

Reply:
Got it, need to define `on_edi_auto_generate` or `auto_handle_generate` events to use the `auto` type rule